xml/registry.rnc

```diff
@@ -62,26 +61,9 @@ Type = element type {
     text
 }
 
-# <groups> defines a group of enum groups
-Groups = element groups {
```

Comment:
I wouldn't think the updated XML would validate against the schema with this stuff removed, while the group tags themselves are marked deprecated but still present?

Reply:
Fixed.